bin/prepare-release.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
```

Comment:
I wish we could write this in something other than bash, but I guess it's still our best option for "cross-platform and always available".  Not a problem with this PR, I guess I just wish the world were different :)

We should probably have @Dexterp37 test this on Windows and/or WSL.  When we were using `bumpversion` for `glean_parser` (which basically serves the exact same purpose as this), it choked on the `git` integration because it would use the `git` from WSL (expecting UNIX line endings) with a checkout from the native Windows `git` (that assumed DOS line endings).  I don't know if there's a general solution to that problem, but maybe there's a way we can help it use the correct `git` or provide a user override.

Reply:
Maybe a good solution here is that I simply set my repo to

```
git config --global core.autocrlf input
git config --global core.eol lf
```

?

Reply:
I could always write it in Rust :)